package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@chamaeleonidae/chmln",
+  "version": "1.0.0-alpha.2",
+  "description": "The official Chameleon JavaScript browser client library",
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/chamaeleonidae/chmln",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/chamaeleonidae/chmln.git"
+  },
+  "bugs": {
+    "url": "https://github.com/chamaeleonidae/chmln/issues"
+  },
+  "files": [
+    "index.js",
+    "license",
+    "package.json",
+    "readme.md"
+  ]
```

Comment:
The process to deploy is quite simple and I didn't add it to the readme, because it's public and it only matters to us... In the console (root folder):

1. npm login (user, pass, 2facode)
2. npm publish --access public

I'll be adding this to a NPMJS page on Notion and this could also be added to scripts in the future, but I really want to start simple and evolve this in baby steps together with the team.